k9mail/src/main/java/com/fsck/k9/view/MessageWebView.java

```diff
@@ -66,6 +72,12 @@ public void configure() {
 
         webSettings.setCacheMode(WebSettings.LOAD_NO_CACHE);
 
+        /* TODO this might improve rendering smoothness when webview is animated into view
```

Comment:
I don't think so. It's a minor performance thing, not really worth adding another issue to the pile